docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -100,61 +115,60 @@ For example:
 ```python file=/concepts/assets/source_asset.py startafter=start_marker endbefore=end_marker
 from dagster import AssetGroup, AssetKey, SourceAsset, asset
 
-my_source_asset = SourceAsset(key=AssetKey("my_source_asset"))
+my_source_asset = SourceAsset(key=AssetKey("a_source_asset"))
 
 
 @asset
-def my_derived_asset(my_source_asset):
-    return my_source_asset + [4]
+def my_derived_asset(a_source_asset):
+    return a_source_asset + [4]
 
 
-asset_group = AssetGroup(assets=[my_derived_asset], source_assets=[my_source_asset])
+asset_group = AssetGroup(assets=[my_derived_asset], source_assets=[a_source_asset])
```

Comment:
This reference here will need to be changed to `my_source_asset` because the asset definition needs to be passed into the `AssetGroup` class